From: Cordell Bloor <cgmb@slerp.xyz>
Date: Fri, 16 Feb 2024 16:58:50 -0700
Subject: use local mathjax

The sphinx.ext.mathjax extension defaults to loading mathjax from a
CDN, which results in the lintian warning 'privacy-breach-generic'.
Use a local copy of mathjax to prevent that problem.

Forwarded: not-needed
---
 docs/source/conf.py | 3 +++
 1 file changed, 3 insertions(+)

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -180,3 +180,6 @@
      author, 'RCCL', 'One line description of project.',
      'Miscellaneous'),
 ]
+
+# use local mathjax for local documentation
+mathjax_path = "/usr/share/javascript/mathjax/MathJax.js"
